tests/unit/packaging/test_services.py

```diff
@@ -13,6 +13,7 @@
 import io
 import os.path
 
+import b2sdk.v2
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏is importing `b2sdk.v2` to access `b2sdk.exception` the correct pattern we should be using?

Reply:
oh hm, that is surprising behavior.  I imported as such thinking I'd need to monkeypatch for test.

seems b2sdk.Exception was imported as a side effect. made explicit in ec47f8dfc89ebde4c1d5a117792ae5d2a261d186

Reply:
`b2sdk.v2.exception` will never change the interface on you, except for an agreed on way (we might make an exception more specific, but then `isinstance` will keep working, though stuff like `repr()` might give a different result)

`b2sdk.exception` will, so if you really need to use that, you have to pin a dependency to a minor version. From what I see you only use the public interface so as long as everything is imported from `b2sdk.v2`, you can pin to `>=1.20.0;<2.0.0`